content/blog/2020/10/VSCode-LanguageSupport-0.0.28/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "New release of VS Code Language Support for Apache Camel 0.0.28"
+date: 2020-10-27
+authors: [apupier]
+categories: ["Releases","Tooling"]
+preview: "New release of VS Code Language Support for Apache Camel 0.0.28: Camel 3.6.0, Camel Kafka Connector and Camel K Modeline and outline"
+---
+
+A new release of [VS Code Language Support for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-apache-camel) 0.0.28 is available. It is providing an interesting set of new features.
```

Comment:
This might be made a bit smoother by editing to: "We have released a new version (x.y.z) of XXXX, which includes some interesting new features."